chatgse/app/constant.ts

```diff
@@ -12,7 +12,7 @@ export const DOCS_URL = 'https://biocypher.org'
 
 export const DEFAULT_CORS_HOST = "https://a.nextweb.fun";
 export const DEFAULT_API_HOST = `${DEFAULT_CORS_HOST}/api/proxy`;
-export const OPENAI_BASE_URL = "https://api.openai.com";
+export const OPENAI_BASE_URL = "http://localhost:5000/";
```

Comment:
Does this interfere with regular behaviour when using the app in non-WASM mode? @xiaoranzhou

Reply:
@xiaoranzhou could you comment?